cluster-autoscaler/metrics/metrics.go

```diff
@@ -125,6 +125,14 @@ var (
 		},
 	)
 
+	maxNodesTotal = k8smetrics.NewGauge(
+		&k8smetrics.GaugeOpts{
+			Namespace: caNamespace,
+			Name:      "max_nodes_total",
```

Comment:
Can you rename to max_nodes_count? IIRC in prometheus metric convention (which CA follows) having '_total' in name is generally expected for counters and gauges should be suffixed with '_count'.
Also this way it will be consistent with nodes_count.

Reply:
Done